src/server/world.zig

```diff
@@ -928,6 +935,37 @@ pub const ServerWorld = struct { // MARK: ServerWorld
 		self.dropWithCooldown(stack, pos, dir, velocity, 0);
 	}
 
+	fn tickBlocksInChunk(self: *ServerWorld, _chunk: ?*chunk.ServerChunk) void {
+		if(_chunk) |ch| {
```

Comment:
Please generally do early returns instead of long if's to avoid the extra indentation:
```suggestion
		const ch = _chunk orelse return;
```
Furthermore I think that this should probably be done at the call site without passing an optional to the function in the first place.